### PR TITLE
[16.0][MIG] animal

### PR DESCRIPTION
**Pull request by Deriman-Alonso**
Standart migration

### CODE REVIEW

File: animal/models/animal.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _order = "name"
+
+    name = fields.Char()
+    ref = fields.Char(string="Reference")
+    species_id = fields.Many2one("animal.species", string="Species", required=True)
+    breed_id = fields.Many2one("animal.breed", string="Breed", required=True)
+    color_id = fields.Many2one("animal.color", string="Color")
+    size = fields.Char()
+    weight = fields.Float(string="Weight (in kg)")
```
Comment (christian-ramos-tecnativa) [inline animal/models/animal.py:18]: Hi @Deriman-Alonso , I think we should take advantage of the opportunity to remove the specification regarding the pet's weight in kilograms. I believe the best approach would be to do something similar to how products/pickings handle weight based on the overall adjustment. 
image
image

**Review by christian-ramos-tecnativa — CHANGES_REQUESTED** [review]

File: animal/models/animal.py (hunk 1 above)
Comment (Deriman-Alonso) [reply animal/models/animal.py:18]: Makes sense to me, I'm working on it 👍.

Comment (Deriman-Alonso) [reply animal/models/animal.py:18]: Hi @Christian-RB , as far as my understanding goes, those UoM settings comes from the "product" module. Would it be properly to make the "animals" module depend on  "product", or rather create its own menu in the settings, even tho it means there could be different types of UoM for the weight?

Comment (christian-ramos-tecnativa) [reply animal/models/animal.py:18]: Oh You are right @Deriman-Alonso, in that case I assume is better to keep it the same way. Thanks!

**Review by christian-ramos-tecnativa — APPROVED** [review]

**Review by jado95 — APPROVED** [review]

**Review by MarcoCalcagni — APPROVED** [review]
Go 

we have in production 

@pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
/ocabot migration animal
/ocabot merge nobump

**Pull request merged**